### DISCUSSION TITLE
Code analyzers

### DISCUSSION THREAD

**Discussion by NN---** [rsdn/CodeJam#132] [General]
@andrewvk @ig-sinicyn Discussion about different analysis rules.

**Comment by NN---** [comment]
What do you think about CLSCompliant ? Is it still relevant today ?
Should we simply mark CLSCompliant(false) for the whole assembly or put on the relevant methods only ?

**Comment by andrewvk** [comment]
Formally, the attribute is not obsolete. But in fact, I strongly doubt that it has any practical sense.

**Comment by NN---** [comment]
The analysis is enabled by default :) https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1014
